docs/perf_analyzer.md

```diff
@@ -453,6 +453,36 @@ By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
 protocol can be specificed with the -i option. If GRPC is selected the
 --streaming option can also be specified for GRPC streaming.
 
+### SSL/TLS Support
+
+perf_analyzer can be used to benchmark Triton service behind SSL/TLS-enabled endpoints. These options can help in establishing secure connection with the endpoint and profile the server.
+
+For gRPC, see the following options:
+
+* `--ssl-grpc-root-certifications-file`
+* `--ssl-grpc-private-key-file`
+* `--ssl-grpc-certificate-chain-file`
+
+More details here: https://grpc.github.io/grpc/cpp/structgrpc_1_1_ssl_credentials_options.html
+
+The [inference protocol gRPC SSL/TLS section](inference_protocols.md#ssltls) describes server-side options to configure SSL/TLS in Triton's gRPC endpoint.
+
+For HTTPS, the following options are exposed:
+
+* `--ssl-http-verify-peer`
```

Comment:
I was thinking.. wouldn't it make more sense to rename all these options as `--ssl-https_*`?  It becomes more clear that these options will be used for https endpoint only.

Reply:
renamed to `--ssl-https-*` 👍